resources/META-INF/plugin.xml

```diff
@@ -13,7 +13,7 @@
 
   <category>Custom Languages</category>
   <version>SNAPSHOT</version>
-  <idea-version since-build="182.5107.16" until-build="192.SNAPSHOT"/>
+  <idea-version since-build="182.5107.16" until-build="191.*"/>
```

Comment:
Is this correct?

Reply:
(I mean, I'm surprised that we're bounding to 191.*.)

Reply:
You're right! I should not have deleted the final build spec. That broke the debugger. Restored. PTAL.